### DISCUSSION TITLE
is turbo engine usable outside of this repo?

### DISCUSSION THREAD

**Discussion by sourcec0de** [vercel/turborepo#2445] [Ideas]
Hey all :wave: 
Really impressive announcements at NextJS Conf.

I'm wondering what the plan is for `turbo-engine`.
From @padmaia's [presentation](https://www.youtube.com/watch?v=btqdaqEHw0A&t=854s), it sounded like the plan is to keep it decoupled so "other tools" could use it.
Is this currently possible today even if the API isn't stable?

**Comment by jridgewell** [comment]
Correct, we plan on publishing to the `turbo-tasks` crate, which provides the core APIs to define tasks and the macros that decorate functions and values. It's still unstable, and isn't well documented yet.

Probably the only way to use it outside this monorepo currently is to point your `Cargo.toml` dependency to this repo at a pinned SHA.

**Comment by sourcec0de** [comment]
@jridgewell thanks for the fast answer.
I'll give that a shot!
